lib/spack/spack/package.py

```diff
@@ -1017,6 +1017,13 @@ def provides(self, vpkg_name):
             for s, constraints in self.provided.items() if s.name == vpkg_name
         )
 
+    def virtuals_provided(self):
```

Comment:
Can we do this:
```suggestion
    @property
    def virtuals_provided(self):
```
to be consistent with other attributes of the same kind?